<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tbkt_Widget</class>
 <widget class="QWidget" name="Tbkt_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>同步课堂</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images/images.qrc">
    <normaloff>:/new/prefix1/Images/logo.png</normaloff>:/new/prefix1/Images/logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>1920</width>
       <height>1080</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1080</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget_webview" native="true">
        <property name="minimumSize">
         <size>
          <width>1920</width>
          <height>1080</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1920</width>
          <height>1080</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_tbkt_tool" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>90</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1920</width>
       <height>90</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget_tbkt_tool
{
        background:rgb(29,85,208);

}
QPushButton {
    border-style: solid;
    border-width: 2px;
    /*border-color:rgb(200, 200, 200);*/
    border-radius: 20px;
;

    background-color:transparent;
    background-repeat: no-repeat;
    background-position: left center;
}

QPushButton::menu-indicator {
    image: url(myindicator.png);
    subcontrol-position: right center;
    subcontrol-origin: padding;
    right: 10px;
    top: 15px;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="6">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>209</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>tips</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_beike">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/beike.png</normaloff>:/new/prefix1/Images/beike.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>55</width>
            <height>55</height>
           </size>
          </property>
          <property name="autoRepeatDelay">
           <number>200</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_keben">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/keben.png</normaloff>:/new/prefix1/Images/keben.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="autoRepeatDelay">
           <number>200</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_dianping">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/dianping.png</normaloff>:/new/prefix1/Images/dianping.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="autoRepeatDelay">
           <number>200</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_weike">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/weike.png</normaloff>:/new/prefix1/Images/weike.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="autoRepeatDelay">
           <number>200</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>208</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="7">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_tiwen">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/tiwen.png</normaloff>:/new/prefix1/Images/tiwen.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="autoRepeatDelay">
           <number>200</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_jishi">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/jishi.png</normaloff>:/new/prefix1/Images/jishi.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="autoRepeatDelay">
           <number>200</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_paihang">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/paihang.png</normaloff>:/new/prefix1/Images/paihang.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="autoRepeatDelay">
           <number>200</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_huabi">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/huabi.png</normaloff>:/new/prefix1/Images/huabi.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="autoRepeatDelay">
           <number>200</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_huaban">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/huaban.png</normaloff>:/new/prefix1/Images/huaban.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="autoRepeatDelay">
           <number>200</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_upan">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/upan.png</normaloff>:/new/prefix1/Images/upan.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="autoRepeatDelay">
           <number>200</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>biaoyang</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>209</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_setting">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/shezhi.png</normaloff>:/new/prefix1/Images/shezhi.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="autoRepeatDelay">
           <number>200</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_dianming">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/jushoudianming.png</normaloff>:/new/prefix1/Images/jushoudianming.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="autoRepeatDelay">
           <number>200</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_qiangda">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/quanyuanqiangda.png</normaloff>:/new/prefix1/Images/quanyuanqiangda.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="autoRepeatDelay">
           <number>200</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_hudong">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/hudongdati.png</normaloff>:/new/prefix1/Images/hudongdati.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="autoRepeatDelay">
           <number>200</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Images/images.qrc"/>
 </resources>
 <connections/>
</ui>
